buildscripts/condarecipe.local/meta.yaml

```diff
@@ -70,7 +71,7 @@ test:
     - ipython                  # [not (armv6l or armv7l or aarch64)]
     - setuptools
     - tbb  >=2021              # [not (armv6l or armv7l or aarch64 or linux32 or ppc64le)]
-    - llvm-openmp              # [osx]
+    - llvm-openmp              # [osx and x86_64]
```

Comment:
This is banning all testing with llvm-openmp on M1? Won't the `run_constrained` just handle this?

Reply:
I guess this comes down to the definition of the "test environment". IIRC in principle the `meta.yaml` should be a description of what is correct and it's up to e.g. `conda build --test` to have the appropriate channels etc to pick up the packages. In scenarios where testing is performed from the installed package directly, it's a question of ensuring that environment is suitable if OpenMP testing is required.